WORKSPACE

```diff
@@ -67,6 +67,16 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "yarn_install"
 node_repositories(
     package_json = ["//:package.json"],
     preserve_symlinks = True,
+    node_version = "10.9.0",
+    yarn_version = "1.9.2",
+    node_repositories = {
```

Comment:
I don't want this to be the example for users, why not just 10.3 which is already available in rules_nodejs

Reply:
For sure. That's not meant to be permanent. I pulled in my angular PR that works with fine grained deps which is based of master which now requires node 10 so I just copied over the config from the angular repo. Not meant to be merged.

Reply:
Oh. Angular master and patch now both require node 10.9 or greater.
```
ERROR: Analysis of target '//src:prodserver' failed; build aborted: no such package '@angular_deps//': yarn_install failed: yarn install v1.3.2
$ node tools/yarn/check-yarn.js
[1/5] Validating package.json...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error angular-srcs@6.1.7: The engine "node" is incompatible with this module. Expected version ">=10.9.0 <11.0.0".
error angular-srcs@6.1.7: The engine "yarn" is incompatible with this module. Expected version ">=1.9.2 <2.0.0".
error Found incompatible module
)
```
Will add that version to rules_nodejs